runtime/src/main/scala/akka/grpc/javadsl/WebHandler.scala

```diff
@@ -66,7 +66,8 @@ object WebHandler {
       mat: Materializer,
       corsSettings: CorsSettings): JFunction[HttpRequest, CompletionStage[HttpResponse]] = {
     import scala.collection.JavaConverters._
-    val servicesHandler = concat(handlers.asScala.toList: _*)
+
+    val servicesHandler = concat(handlers.asScala: _*)
```

Comment:
this `.toList` was actually needed for compiling on Scala 2.13

Reply:
oh, thanks for pointing out